packages/invoices/src/OutboundShipment/api/hooks/line/useOutboundAllocateSelectedLines.ts

```diff
@@ -67,12 +67,45 @@ export const useOutboundAllocateSelectedLines = (): {
       return;
     }
 
-    const response = await mutateAsync(selectedUnallocatedLines);
-    const { allocateOutboundShipmentUnallocatedLines } = response;
-    const number = allocateOutboundShipmentUnallocatedLines?.length;
-    const onSuccess = success(t('messages.allocated-lines', { number }));
-    onSuccess();
-  };
+    const batchResponse = await mutateAsync(selectedUnallocatedLines);
+
+    if (batchResponse?.__typename === 'BatchOutboundShipmentResponse') {
+      const { allocateOutboundShipmentUnallocatedLines } = batchResponse;
+      const count = {
+        success: 0,
+        partial: 0,
+        failed: 0,
+      };
+
+      allocateOutboundShipmentUnallocatedLines?.forEach(line => {
+        const { id, response } = line;
+        if (
+          response?.__typename === 'AllocateOutboundShipmentUnallocatedLineNode'
+        ) {
+          if (response?.deletes.some(({ id: deleted }) => id === deleted)) {
+            count.success++;
+            return;
+          }
+          if (response.inserts.totalCount > 0) {
+            count.partial++;
+            return;
+          }
+          count.failed++;
+        }
+      });
 
+      if (count.success > 0) {
```

Comment:
The request was to show full, partial and failed - but we don't have much space in a snackbar, so I went with this approach. Most likely users will have only one type, so I thought this would work. We could go all out and have a summary modal, but I prefer the less invasive snackbar. you don't have to click to clear the snack, like you do a modal.